dll/ntdll/ldr/ldrinit.c

```diff
@@ -86,7 +86,12 @@ ULONG LdrpActiveUnloadCount;
 //extern LIST_ENTRY RtlCriticalSectionList;
 
 VOID NTAPI RtlpInitializeVectoredExceptionHandling(VOID);
-VOID NTAPI RtlpInitDeferedCriticalSection(VOID);
+
+#if (NTDDI_VERSION < NTDDI_WS03)
+#define RtlpInitDeferredCriticalSection RtlpInitDeferedCriticalSection
```

Comment:
If you define both and have one just call the other there's no need for an ifdef.

Reply:
Why is this needed at all? It's an internal function, not exported anywhere.